src/config/pollinator.ts

```diff
@@ -0,0 +1,5 @@
+export const STAGE_POLLINATOR_JIRA_HOST = "https://fusion-arc-pollinator-staging-app.atlassian.net";
```

Comment:
Given GH4J is an open-source project, I don't think we should hardcore Atlassian's internals. IMHO this should be controlled by a feature flag...